service/openapi.yaml

```diff
@@ -366,4 +375,16 @@ components:
           examples:
             - 60000
       required:
-        - name
\ No newline at end of file
+        - name
+    featureCaching:
+      type: object
+      description: Configuration of the caching mechanism in the provider (used by server providers)
+      properties:
+        enabled:
+          type: boolean
+          description: set to true if you want the provider to cache the evaluation results
+        ttl:
```

Comment:
Default `TTL` can be 0 and we will relay only on the polling to evict data from the cache.

Reply:
Agree, by default the behaviour should be no-cache so every evaluation is done remotely.